browser/flag_descriptions.h

```diff
@@ -0,0 +1,95 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_FLAG_DESCRIPTIONS_H_
+#define BRAVE_BROWSER_FLAG_DESCRIPTIONS_H_
+
+#include "brave/components/brave_rewards/common/buildflags/buildflags.h"
+
+namespace flag_descriptions {
+
+extern const char kBraveAdblockCnameUncloakingName[];
```

Comment:
why not put constexpr strings right here and remove ```flag_descriptions.cc```? I don't see a reason why we should keep .cc file for this.

Reply:
Good point. Fixed that in the latest push, thanks!